README.md

```diff
@@ -134,6 +134,8 @@ TENANT_ID=
 SUBSCRIPTION_ID=
 ACR_NAME=
 ACR_PASSWORD=
+ORGANIZATION=https://dev.azure.com/your-organization

```

Comment:
Hey @robece there is no need to add `organization` and `project` on every `az` command. if you run the following command to configure azure devops CLI: `az devops configure --defaults organization=https://dev.azure.com/your-organization project=YourProject`. This command is already on the `Getting Started` section.